CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Raised minimum required PHP version from `7.2` to `7.3`.
 - Upgraded Symfony framework version from `4` to `5`.
 - Changed `user-ids` and `line-item-ids` input options of [roster:doctrine-result-cache:warmup](docs/cli/doctrine-result-cache-warmer-command.md) command to `usernames` and `line-item-slugs`.
+- Application namespace now is `OAT\SimpleRoster` instead of `App`
```

Comment:
Could you please rephrase it as a sentence, like the other changes?

Reply:
About this comment. I do not know if this is enough. Please let me know in case it's not ok.